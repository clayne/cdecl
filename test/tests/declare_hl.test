cdecl @ /dev/null @ @ declare x as short long @ 65
