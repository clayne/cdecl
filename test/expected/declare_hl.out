declare x as short long
        ^
9: error: "short long int" is illegal for built-in type
